exercises/pig-latin/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "pig-latin"
+version = "0.1.0"
+authors = ["sacherjj <sacherjj@gmail.com>"]
+
+[dependencies]
+regex = "0.2"
```

Comment:
since it is not necessarily the case that any student's solution will use either `regex` or `lazy_static`, could you move these dependencies to a file named `Cargo-example.toml`? See https://github.com/exercism/rust/blob/master/exercises/alphametics/Cargo-example.toml versus https://github.com/exercism/rust/blob/master/exercises/alphametics/Cargo.toml for an example. Otherwise, a student may see these dependencies and not know what to do with them, or think that using them is mandatory and that would unintentionally narrow the kinds of solutions we might see submitted.